### DISCUSSION TITLE
Warping in c#

### DISCUSSION THREAD

**Discussion by Alex802c** [SixLabors/ImageSharp#2211] [Q&A]
So I have an image i would like to warp. I've been snooping around in the different options. I cant seem to find what I am looking for. 

Does anyone have experience with this and know how to warp an image.

**Comment by dlemstra** [comment]
Can you explain what you mean by wrapping? Is that something similar to this: https://dlemstra.github.io/magick-wasm-docs/classes/magick-image/liquidRescale?

**Comment by JimBobSquarePants** [comment]
We do have perspective and affine transforms [available](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Processing.TransformExtensions.html) if that's what you mean?
